shardingsphere-jdbc/shardingsphere-jdbc-core/src/test/java/org/apache/shardingsphere/driver/jdbc/core/statement/ReadwriteSplittingPreparedStatementTest.java

```diff
@@ -88,7 +87,6 @@ public void assertGetGeneratedKeysWithPrimaryKeyIsNull() throws SQLException {
             int columnCount = generatedKeys.getMetaData().getColumnCount();
             for (int index = 0; index < columnCount; index++) {
                 assertNotNull(generatedKeys.getObject(index + 1));
-                assertNotNull(generatedKeys.getMetaData().getColumnLabel(index + 1));
```

Comment:
Refer to https://github.com/apache/shardingsphere/pull/19878#issuecomment-1207161251 .